kubectl-apply-all.sh

```diff
@@ -12,4 +13,10 @@
 #
 # Apply the base Soucegraph deployment
 # shellcheck disable=SC2068
+
+# Run the DB migration job and wait for it to complete
+kubectl delete -f base/migrator/migrator.Job.yaml --ignore-not-found=true
+kubectl apply -f base/migrator/migrator.Job.yaml
+kubectl wait -f base/migrator/migrator.job.yaml --for=condition=complete --timeout=-1s
```

Comment:
I would propose to add comment here for customers, b/c `-1s` sounds scary to me :) They should be aware this might take long - although I know that they have docs.